src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -2204,7 +2202,7 @@ public FilePreferences getFilePreferences() {
                 // We choose the data directory, because a ".bak" file should survive cache cleanups
                 getPath(BACKUP_DIRECTORY, OS.getNativeDesktop().getBackupDirectory()),
                 getBoolean(CONFIRM_LINKED_FILE_DELETE),
-                getBoolean(TRASH_INSTEAD_OF_DELETE));
+                getBoolean(TRASH_INSTEAD_OF_DELETE, OS.getNativeDesktop().moveToTrashSupported()));
```

Comment:
This is wrong. Can this be enabled in the preferences even though it is not supported?

Reply:
Was already wrong before the chris change. 😉 isSupported is logic, not a preferences option.

Reply:
<img width="525" alt="grafik" src="https://github.com/JabRef/jabref/assets/320228/947f9181-3fea-4342-bb33-89d39c1f77d6">

Reply:
https://github.com/JabRef/jabref/blob/cfabd9fc6e0373e7a66b1702091cab1f4ce1a879/src/main/java/org/jabref/gui/preferences/linkedfiles/LinkedFilesTab.java#L67

Reply:
Nevertheless, the preference should not be true if it is not supported system-wide. I thought, in https://github.com/JabRef/jabref/pull/10591 I implemented it as: "default = supported by OS" and then "disabled/enabled = supported by OS". 

Maybe an additional line:

```java
if (!OS.getNativeDesktop().moveToTrashSupported()) {
  moveToTrash.setValue(false)
}
```

to ensure that it is both disabled and false if trash is not supported.

Reply:
It was in defaults. but this broke the system menu bar in macOS. This is why i removed it in defaults and just added it as fallback value

Reply:
Which part exactly broke the menu bar? The issue did not show any exception, so I cannot understand the root cause.

Reply:
My suggestion would be just to make true the default, but it's greyed out in the preferences and where moveToTrash is used, there is an additional check, if this is supported. Check could go to JabRefDesktop#moveToTrash or sthg and then delete instead.